uat/testing-features/src/main/java/com/aws/greengrass/steps/NetworkUtilsSteps.java

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
these steps should be in OTF, not here.

Reply:
PR for OTF  is [here](https://github.com/aws-greengrass/aws-greengrass-testing/pull/215)